docs/guides/initialize-state-with-props.md

```diff
@@ -0,0 +1,126 @@
+---
+title: initialize-state-with-props
```

Comment:
Can we make this sentence case to make the rest? 
Can we also uncomment the line below and give it a number

Reply:
> Can we make this sentence case to match the rest?

Is the title okay besides the format? Should it mention of context?

> Can we also uncomment the line below and give it a number

I wasn't sure what number this refers to. What should it be?

Reply:
I think the rest of the title is good 👍 

The number is the overall position in the left hand nav. Unfortunately, it's a bit manual, but find the page you want this page to appear after, grab increment the nav number by one. You should then update any subsequent nav numbers (this is the unfortunate bit, I'm not sure if there is an alternative in the core website atm)

Reply:
I would like to put it at the bottom of the "GUIDES" section, under "How to reset state", but it's unclear how the nav number correlates to the entry's position on the site. I've put together an excalidraw sketch highlighting what I mean. 
The numbers on the left are the nav numbers in each doc entry. The numbers on the right are how the nav appears to be ordered. 

![zustand-pr-1291-navs excalidraw](https://user-images.githubusercontent.com/92656523/190057887-ef8ee0f9-1e72-47cd-8e7e-c8711e431e83.png)

Reply:
I'm not super familiar with the core website, but you can checkout the ordering logic here: 

https://github.com/pmndrs/website/blob/docs/src/utils/docs.ts#L144

I'm not sure why that #15 isn't bugging out, I wonder if it has something to do with the way pages are grouped into parents... 

[react three fiber](https://github.com/pmndrs/react-three-fiber/tree/master/docs) also use the same website for their docs, and they seem to follow an order nav count (similar to the above without the mistakes you highlighted). 

I will try to go through an audit the nav numbers for Zustand, unless you want to do it while you're in there and adding this page?

Reply:
> I will try to go through an audit the nav numbers for Zustand, unless you want to do it while you're in there and adding this page?

Yes I'd like to do that 😁. Just to clarify one more time, the correct order is on the right side of the image?

Reply:
Yep, looks good to me, except, keep "createContext from zustand/context" for now. So, the final two numbers down the right should be 18 + 19